internal/cmd/local/command_kafka_topic_consume.go

```diff
@@ -0,0 +1,163 @@
+package local
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/uuid"
+	"github.com/spf13/cobra"
+
+	ckafka "github.com/confluentinc/confluent-kafka-go/kafka"
+
+	"github.com/confluentinc/cli/internal/cmd/kafka"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/log"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+func (c *command) newConsumeCommand() *cobra.Command {
```

Comment:
Since `command` exists at the top-level, we should format like this to avoid naming conflicts in the future.
```suggestion
func (c *command) newKafkaTopicConsumeCommand() *cobra.Command {
```